src/test/java/se/sundsvall/casemanagement/integration/casedata/CaseDataServiceTest.java

```diff
@@ -241,6 +240,17 @@ void testGetStatusNotFound() {
 			.hasFieldOrPropertyWithValue("detail", Constants.ERR_MSG_STATUS_NOT_FOUND);
 	}
 
+	@Test
+	void patchErrandWithAttachmentNotFound() {
+		final var id = UUID.randomUUID().toString();
+		when(caseDataClientMock.postAttachment(any())).thenThrow(Problem.valueOf(Status.NOT_FOUND));
+
+		assertThatThrownBy(() -> caseDataService.patchErrandWithAttachment(id, List.of(new AttachmentDTO())))
```

Comment:
Refactor the code of the lambda to have only one invocation possibly throwing a runtime exception.